prow/cluster/monitoring/BUILD.bazel

```diff
@@ -48,15 +51,30 @@ k8s_object(
     template = "@operator_framework_community_operators//:upstream-community-operators/prometheus/servicemonitor.crd.yaml",
 )
 
+k8s_object(
+    name = "prow_prometheusrule",
+    cluster = "{STABLE_PROW_CLUSTER}",
+    kind = None,
+    template = "//prow/cluster/monitoring/mixins/prometheus:prow_prometheusrule",
+)
+
 load("@io_bazel_rules_k8s//k8s:objects.bzl", "k8s_objects")
 
 k8s_objects(
-    name = "prometheus_operator_crd",
+    name = "prow_monitoring_objects",
     objects = [
         ":alertmanager_crd",
         ":prometheus_crd",
         ":prometheusrule_crd",
         ":servicemonitor_crd",
+        "//prow/cluster/monitoring/mixins/prometheus_out:prow_prometheusrule",
```

Comment:
Why are these changes in this PR?

Reply:
`prow_prometheusrule` is the actually for the alerts' defs.